app/workflows/lists/conversion/voluntary/sections/kickoff.yml

```diff
@@ -38,7 +38,8 @@ tasks:
     title: External stakeholder kick-off
     hint:
       You should start this work after you've had the handover with the regional
-      delivery officer.
+      delivery officer. This is a good opportunity for you to talk to about
```

Comment:
See comment for this same line within the involuntary kickoff.yml - same applies here too